engine-rest/engine-rest-openapi/src/main/templates/paths/history/decision-requirements-definition/{id}/statistics/get.ftl

```diff
@@ -0,0 +1,73 @@
+<#-- Generated From File: camunda-docs-manual/public/reference/rest/history/decision-requirements-definition/get-statistics/index.html -->
+<#macro endpoint_macro docsUrl="">
+{
+  <@lib.endpointInfo
+      id = "getHistoricDecisionRequirementsDefinitionStatistics"
+      tag = "Historic Decision Requirements Definition"
+      summary = "Get DRD Statistics"
+      desc = "Retrieves evaluation statistics of a given decision requirements definition."
+  />
+
+  "parameters" : [
+
+      <@lib.parameter
+          name = "id"
+          location = "path"
+          type = "string"
+          required = true
+          desc = "The id of the decision requirements definition."
+      />
+
+      <@lib.parameter
+          name = "decisionInstanceId"
+          location = "query"
+          type = "string"
+          desc = "Restrict query results to be based only on specific evaluation
+                  instance of a given decision requirements definition."
+          last = true
+      />
+
+  ],
+
+  "responses": {
+
+    <@lib.response
+        code = "200"
+        dto = "DecisionRequirementsDefinitionStatisticsDto"
+        array = true
+        desc = "Request successful."
+        examples = ['"example-1": {
+                       "summary": "request",
```

Comment:
We could use something like "request without decisionInstanceId" here. But that might be too long.

Reply:
I would rather stick with one basic `request` and then have other examples that state how they differ from that than encoding that in the name of the basic request as well.

Reply:
Ok, thats also fine.